battleship-service/src/main/java/dreamteam/battleship/service/springcontroller/model/response/Restart.java

```diff
@@ -0,0 +1,7 @@
+package dreamteam.battleship.service.springcontroller.model.response;
+
+/**
+ * The class is to return the information about the restart
```

Comment:
The class is redundant on class Javadoc. Everybody knows it's a class. Remove please. Also, class does not return.